Makefile-rpm-ostree.am

```diff
@@ -112,16 +102,18 @@ if BUILDOPT_ENABLE_SQLITE_RPMDB_DEFAULT
 cargo_build += --features sqlite-rpmdb-default
 endif
 
-librpmostree_rust_path = $(top_srcdir)/target/@RUST_TARGET_SUBDIR@/librpmostree_rust.a
 # If the target directory exists, and isn't owned by our uid, then
 # we exit with a fatal error, since someone probably did `make && sudo make install`,
 # and in this case cargo will download into ~/.root which we don't want.
 LIBRPMOSTREE_RUST_SRCS = $(shell find rust/src/ -name '*.rs') Cargo.toml Cargo.lock cbindgen.toml
 # FIXME - build all this code in a rpmostree-sys crate, or just move all the C/C++ build
 # to Rust.  Currently this forces build system serialization
-$(librpmostree_rust_path): Makefile $(LIBRPMOSTREE_RUST_SRCS) librpmostreeinternals.la
+rpm-ostree: Makefile $(LIBRPMOSTREE_RUST_SRCS) librpmostreeinternals.la
 	$(cargo_build) $(CARGO_RELEASE_ARGS)
+	ln -sfr target/release/rpm-ostree $@
```

Comment:
This assumes a release build. Should be conditional on `RUST_DEBUG`, no? (I personally always want to build C code with `-g -Og` and Rust code in debug mode by default.)

Reply:
OK tweaked, though remember now you need to use e.g. `env CFLAGS='-Og' CXXFLAGS='-Og' ./configure ...` to turn it on.